packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -365,12 +365,12 @@ describe("calcite-list", () => {
     const list = await page.find("calcite-list");
     await page.waitForTimeout(listDebounceTimeout);
 
-    expect(await list.getProperty("filteredItems")).toHaveLength(3);
-    expect(await list.getProperty("filteredData")).toHaveLength(3);
+    expect(await list.getProperty("filteredItems")).toHaveLength(2);
```

Comment:
Should this have changed? Filtering should behave the same when `matchFields` are not specified, right?